[debug] [2024-11-07T21:03:14.932Z] ----------------------------------------------------------------------
[debug] [2024-11-07T21:03:14.936Z] Command:       /home/kell/.nvm/versions/node/v18.15.0/bin/node /home/kell/.nvm/versions/node/v18.15.0/bin/firebase login
[debug] [2024-11-07T21:03:14.937Z] CLI Version:   13.24.1
[debug] [2024-11-07T21:03:14.937Z] Platform:      linux
[debug] [2024-11-07T21:03:14.937Z] Node Version:  v18.15.0
[debug] [2024-11-07T21:03:14.938Z] Time:          Thu Nov 07 2024 17:03:14 GMT-0400 (Horário Padrão do Amazonas)
[debug] [2024-11-07T21:03:14.938Z] ----------------------------------------------------------------------
[debug] 
[info] Already logged in as kcristhianesilva@gmail.com
[debug] [2024-11-07T21:03:25.140Z] Error: Timed out.
    at Timeout._onTimeout (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:04:22.157Z] ----------------------------------------------------------------------
[debug] [2024-11-07T21:04:22.160Z] Command:       /home/kell/.nvm/versions/node/v18.15.0/bin/node /home/kell/.nvm/versions/node/v18.15.0/bin/firebase init
[debug] [2024-11-07T21:04:22.161Z] CLI Version:   13.24.1
[debug] [2024-11-07T21:04:22.161Z] Platform:      linux
[debug] [2024-11-07T21:04:22.161Z] Node Version:  v18.15.0
[debug] [2024-11-07T21:04:22.162Z] Time:          Thu Nov 07 2024 17:04:22 GMT-0400 (Horário Padrão do Amazonas)
[debug] [2024-11-07T21:04:22.162Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-11-07T21:04:22.280Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-11-07T21:04:22.280Z] > authorizing via signed-in user (kcristhianesilva@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /home/kell/portfolio/book-exchange-platform

Before we get started, keep in mind:

  * You are initializing within an existing Firebase project directory

[info] 
=== Project Setup
[info] 
[info] First, let's associate this project directory with a Firebase project.
[info] You can create multiple project aliases by running firebase use --add, 
[info] but for now we'll just set up a default project.
[info] 
[debug] [2024-11-07T21:04:49.451Z] Using project from CLI flag: book-exchange-samsara
[debug] [2024-11-07T21:04:49.455Z] Checked if tokens are valid: true, expires at: 1731015092029
[debug] [2024-11-07T21:04:49.456Z] Checked if tokens are valid: true, expires at: 1731015092029
[debug] [2024-11-07T21:04:49.461Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/book-exchange-samsara [none]
[debug] [2024-11-07T21:05:03.161Z] *** [apiv2] error from fetch(https://firebase.googleapis.com/v1beta1/projects/book-exchange-samsara, {"headers":{},"method":"GET","signal":{}}): FetchError: request to https://firebase.googleapis.com/v1beta1/projects/book-exchange-samsara failed, reason: getaddrinfo ENOTFOUND firebase.googleapis.com
[debug] [2024-11-07T21:05:03.162Z] Failed to make request to https://firebase.googleapis.com/v1beta1/projects/book-exchange-samsara (original: request to https://firebase.googleapis.com/v1beta1/projects/book-exchange-samsara failed, reason: getaddrinfo ENOTFOUND firebase.googleapis.com)
[debug] [2024-11-07T21:05:08.172Z] Error: Timed out.
    at Timeout._onTimeout (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:21.742Z] ----------------------------------------------------------------------
[debug] [2024-11-07T21:05:21.746Z] Command:       /home/kell/.nvm/versions/node/v18.15.0/bin/node /home/kell/.nvm/versions/node/v18.15.0/bin/firebase init
[debug] [2024-11-07T21:05:21.747Z] CLI Version:   13.24.1
[debug] [2024-11-07T21:05:21.747Z] Platform:      linux
[debug] [2024-11-07T21:05:21.747Z] Node Version:  v18.15.0
[debug] [2024-11-07T21:05:21.748Z] Time:          Thu Nov 07 2024 17:05:21 GMT-0400 (Horário Padrão do Amazonas)
[debug] [2024-11-07T21:05:21.748Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-11-07T21:05:21.866Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-11-07T21:05:21.867Z] > authorizing via signed-in user (kcristhianesilva@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /home/kell/portfolio/book-exchange-platform

Before we get started, keep in mind:

  * You are initializing within an existing Firebase project directory

[debug] [2024-11-07T21:05:33.117Z] Error: setRawMode EIO
    at ReadStream.setRawMode (node:tty:77:31)
    at [_setRawMode] [as _setRawMode] (node:internal/readline/interface:380:18)
    at Interface.close (node:internal/readline/interface:530:24)
    at ReadStream.ontermend (node:internal/readline/interface:266:10)
    at ReadStream.emit (node:events:525:35)
    at ReadStream.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:33.127Z] Error: write EIO
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:930:11)
    at Socket._write (node:net:942:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at Console.log (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston/lib/winston/transports/console.js:87:23)
    at Console._write (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston-transport/modern.js:103:17)
    at writeOrBuffer (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:33.130Z] Error: write EIO
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:930:11)
    at Socket._write (node:net:942:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at Console.log (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston/lib/winston/transports/console.js:87:23)
    at Console._write (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston-transport/modern.js:103:17)
    at writeOrBuffer (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:33.132Z] Error: write EIO
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:930:11)
    at Socket._write (node:net:942:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at Console.log (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston/lib/winston/transports/console.js:87:23)
    at Console._write (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston-transport/modern.js:103:17)
    at writeOrBuffer (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:33.135Z] Error: write EIO
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:930:11)
    at Socket._write (node:net:942:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at Console.log (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston/lib/winston/transports/console.js:87:23)
    at Console._write (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston-transport/modern.js:103:17)
    at writeOrBuffer (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:33.138Z] Error: write EIO
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:930:11)
    at Socket._write (node:net:942:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at Console.log (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston/lib/winston/transports/console.js:87:23)
    at Console._write (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston-transport/modern.js:103:17)
    at writeOrBuffer (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:33.140Z] Error: write EIO
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:930:11)
    at Socket._write (node:net:942:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at Console.log (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston/lib/winston/transports/console.js:87:23)
    at Console._write (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston-transport/modern.js:103:17)
    at writeOrBuffer (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-11-07T21:05:33.141Z] Error: write EIO
    at afterWriteDispatched (node:internal/stream_base_commons:160:15)
    at writeGeneric (node:internal/stream_base_commons:151:3)
    at Socket._writeGeneric (node:net:930:11)
    at Socket._write (node:net:942:8)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at Writable.write (node:internal/streams/writable:337:10)
    at Console.log (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston/lib/winston/transports/console.js:87:23)
    at Console._write (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/winston-transport/modern.js:103:17)
    at writeOrBuffer (/home/kell/.nvm/versions/node/v18.15.0/lib/node_modules/firebase-tools/node_modules/readable-stream/lib/internal/streams/writable.js:334:12)
[error] 
[error] Error: An unexpected error has occurred.
